### PR TITLE
imodeljs-native 3.6.6

### PR DESCRIPTION
**Pull request by aruniverse**
* Add InstanceKeysSelectQuery to node key; update tests

* imodeljs-native 3.6.6:
  - https://github.com/iTwin/imodel-native/compare/v3.6.4...v3.6.6

### CODE REVIEW

File: presentation/common/src/presentation-common/hierarchy/Key.ts [hunk 1]
```diff
@@ -27,70 +27,58 @@ export enum StandardNodeTypes {
 export type NodeKey = BaseNodeKey | ECInstancesNodeKey | ECClassGroupingNodeKey | ECPropertyGroupingNodeKey | LabelGroupingNodeKey;
 /** @public */
 export namespace NodeKey { // eslint-disable-line @typescript-eslint/no-redeclare
-  /**
-   * Serialize given [[NodeKey]] to JSON
-   * @deprecated in 3.x. Use [[NodeKey]].
```
Comment (aruniverse) [inline presentation/common/src/presentation-common/hierarchy/Key.ts:32]: need to re-add these deprecation comments

**Review by tcobbs-bentley — APPROVED** [review]
I don't fully understand what this is doing, but bearing that in mind, I don't see anything that stands out as wrong.

**Review by ben-polinsky — APPROVED** [review]

**Review by grigasp — APPROVED** [review]
Presentation changes look fine.

**Comment by pmconne** [comment]
@grigasp @samueljibin please see [failing presentation tests](https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=1918524&view=logs&j=1cf7e48e-5a70-5ad1-a17e-d1bcf9fd77a3&t=1ecafb62-f0f4-5e1b-39e5-8d27cf3bc60b).
Maybe some snaps weren't updated?

**Comment by grigasp** [comment]
> please see [failing presentation tests](https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=1918524&view=logs&j=1cf7e48e-5a70-5ad1-a17e-d1bcf9fd77a3&t=1ecafb62-f0f4-5e1b-39e5-8d27cf3bc60b).
> Maybe some snaps weren't updated?

Looking into this.

**Comment by grigasp** [comment]
apologies to everyone - one of the test failures reveal an issue in the addon, I'm working on a fix and will post a link to the PR shortly

**Comment by grigasp** [comment]
https://github.com/iTwin/imodel-native/pull/128

**Comment by aruniverse** [comment]
@iTwin/itwinjs-core-presentation still seeing failures in presentation-fullstack, should i update the snapshots or?

**Comment by aruniverse** [comment]
> @iTwin/itwinjs-core-presentation still seeing failures in presentation-fullstack, should i update the snapshots or?

If we don't want to update snapshots, and need to fix something else we can drop this last commit

**Comment by grigasp** [comment]
> If we don't want to update snapshots, and need to fix something else we can drop this last commit

the update looks fine, thanks

**Review by pmconne — APPROVED** [review]

**Pull request merged**